app/3.0/docs/tools/polymer-cli-commands.md

```diff
@@ -299,22 +259,26 @@ Compiler options. Valid values are `"auto"`, `"always"` and `"never"`. Defaults
 
 ### --component-dir, -c
 
-The component directory to use. When `--npm` is true, defaults to `"node_modules"`. 
+The component directory to use. When `--npm` is true, defaults to `"node_modules"`. Without the `--npm` flag, defaults to `"bower_components"`. 
```

Comment:
This is useful for 1.x and 2.x element projects, which use ../ URLs to access peer dependencies. 3.x projects should use names to access peer dependencies, so they don't require the same path rewriting as 1.x and 2.x element projects.